Makefile

```diff
@@ -146,10 +146,15 @@ $(controller_gen_targets): $(GO_ENV_REQUISITES) hack/tools/boilerplate.go.txt ha
 	touch -- '$@'
 
 # Run register-gen for each API group version.
+# Usually register-gen should should be:
```

Comment:
Please check this. I think this is a reasonably good option given the current circumstances.

Reply:
Urgh, so this is what we get for not evaluating the upstream alpha and beta releases. Pretty sure we could've avoided this to land in the final Kubernetes release. WDYT concerning https://github.com/kubernetes/kubernetes/issues/129290#issuecomment-2552233989? Should we maybe set those GODEBUG flags, as well?

Reply:
I wonder if we should have this as a separate commit, just for reference, and to maybe revert it whenever the issue is fixed upstream.